data/paver/skeleton/templates/index.html.template

```diff
@@ -1,9 +1,9 @@
-<!doctype html>
+{% raw %}<!doctype html>
 <html>
-    <head><title>$(( plugin ))$ skeleton page</title></head>
+    <head><title>{{ plugin }} skeleton page</title></head>
```

Comment:
I am of the opinion that the default tags should be changed for Angular, rather than changing the Jinja2 default tags. Using default Jinja2 tags and custom Angular tags is a more flexible design.

We could potentially make Angular tags a configurable option, allowing developers to specify their own open/close tags. It would be much harder to make the Jinja2 tags configurable, as the templates are rendered on the server. So we'd have to render them once to insert the correct open/close tags, then have Jinja2 render them to insert proper values.